aladdin-container.sh

```diff
@@ -161,6 +168,61 @@ function _add_authentication_user_to_kubeconfig() {
 EOT
 }
 
+function _handle_aws_config() {
```

Comment:
this function got kind of hairy - any feedback welcome, especially on variable naming (for all the values pulled from config)

Reply:
haha, yeah, it looks ugly but once i read through it all, it makes sense. It's unfortunate but necessary. The names all make sense to me